src/components/profile/UserProfilePage/style.module.scss

```diff
@@ -37,9 +37,10 @@
       }
 
       .profilePic {
-        border-radius: 4rem;
+        border-radius: 50%;
         flex-shrink: 0;
         height: 7rem;
+        object-fit: cover;
```

Comment:
![Screenshot 2024-01-20 at 8 53 41 AM](https://github.com/acmucsd/membership-portal-ui-v2/assets/27360825/c3cdd7c5-a389-4165-8a62-b369223f43be)
It doesn't look very good and profile pics on v1 are following `object-fit: cover`, so I think this'll be more consistent.

Additionally, since we have cropping in v2, new profile pics should fit perfectly anyways so this shouldn't be an issue going forward